src/js/modules/users/components/ContributorPage.js

```diff
@@ -1,49 +1,85 @@
 import React from "react";
 import { connect } from "react-redux";
-import { Link } from "react-router-dom";
 import injectSheet from "react-jss";
+import { Grid, Row, Col } from "react-bootstrap/lib";
 
 import { getContributorFromSlug } from "../selectors";
 import { getArticlesByContributor } from "../../articles/selectors";
 import { getSections } from "../../sections/selectors";
+import { ArticleRow } from "../../articles/components";
```

Comment:
`ArticleList` takes in the prop `section`. However, a user can write articles for multiple sections so by using `ArticleRow`, I can specify which section belongs to which article.